uahf-technical-spec.md

```diff
@@ -199,6 +199,13 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-7 Difficulty adjustement in case of hashrate drop
+
+In case the MTP of the tip of the chain is 12h after the MTP 6 block before,
+the proof of work target is increased by 1/4, which correspond to a reduction
```

Comment:
Suggest to write out 1/4 as "a quarter (1/4)" or "a quarter (25%)". Just the 1/4 can easily misprint or be misread.

Also, it should be "corresponds" here .